WallsAndHoles/newmapdialog.h

```diff
@@ -24,10 +25,21 @@ class NewMapDialog : public QDialog
     ~NewMapDialog();
 
     NewMapData result;
+    Properties TileMapProperty;
+    //Checks if at least one property is selected
+    bool oneChecked;
```

Comment:
This doesn't feel like a very elegant solution. Probably, one should be check by default. In general, it probably shouldn't be the case that any have to be checked.